launchable/commands/subset.py

```diff
@@ -157,9 +157,9 @@ def default_path_builder(file_name):
                     return file_name
                 path_builder = default_path_builder
 
-            for t in glob.iglob(join(base, pattern), recursive=True):
-                if path_builder:
-                    path = path_builder(t[len(base)+1:])
+            for b in glob.iglob(base):
+                for t in glob.iglob(join(b, pattern), recursive=True):
+                    path = path_builder(t[len(b)+1:])  # type: ignore
```

Comment:
Why did you put `type: ignore`? Although I assume `path_builder` is not nullable in this line, you should put a type guard such as `if path_builder is not None` or refactor code itself to prepend destruction in the future.

Reply:
> Although I assume path_builder is not nullable in this line

I think so. because in [line#148](https://github.com/launchableinc/cli/pull/160/files#diff-4c62e35fb4e3b2b6bff8593107cdddd1fd47eff90293edc9d838c9b2f3cff94aR148) define path_builder if it is None.
`pipenv run type` failed. so I wrote `# type: ignore`

but I'll try an another way to solve it.